configs/grumphp-extension.yml

```diff
@@ -37,75 +38,84 @@ grumphp:
         - "\\$_[A-Z_]+\\["
       regexp_type: E
       triggered_by:
-        - php
         - inc
         - install
         - module
-        - theme
+        - php
         - profile
+        - theme
     git_branch_name:
       whitelist:
-        - "#^\\d+\\(.\\d+)?\\.x|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
+        - "#^\\d+(.\\d+)?\\.x|master|develop|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
     git_commit_message:
-      enforce_no_subject_trailing_period: false
-      max_body_width: 80
-      max_subject_width: 80
       matchers:
-        - "/^([A-Z][A-Z0-9]+-\\d+(, [A-Z][A-Z0-9]+-\\d+)*: )?(Add|Change|Fix|Update|Remove) /"
+        - "/^([A-Z][A-Z0-9]+-\\d+(, [A-Z][A-Z0-9]+-\\d+)*: )?(Add|Change|Fix|Update|Remove|Refactor) /"
       case_insensitive: false
     phpcpd:
       exclude:
         - tests
         - vendor
+        - "*.api.php"
+        - "*Test.php"
+        - "*TestBase.php"
+        - "*TestCase.php"
       min_lines: 10
       triggered_by:
-        - php
         - inc
         - install
         - module
-        - theme
+        - php
         - profile
+        - theme
     phpcs:
       standard:
-        - .phpcs.qa-drupal.xml
+        - phpcs.qa-drupal.xml
       report_width: 120
       ignore_patterns:
+        - build/
         - node_modules/
-        - "#^vendor/#"
+        - vendor/
+        - "*.md"
       triggered_by:
-        - php
+        - css
         - inc
         - install
+        - js
         - module
-        - theme
+        - php
         - profile
+        - theme
+        - twig
         - yml
     phpmd:
       ruleset:
-        - .phpmd.qa-drupal.xml
+        - phpmd.qa-drupal.xml
       exclude:
-        - vendor
+        - "build/*"
+        - "tests/*"
```

Comment:
Zou je dat ook niet op tests runnen?

Reply:
Tests can be messy and may contain code duplication...